src/utils/format.ts

```diff
@@ -81,3 +81,96 @@ export const formatCode = (code: string): Promise<string> => {
     semi: config.semi,
   });
 };
+
+/**
+ * Formats a SQLite statement by adding proper indentation and line breaks.
+ * Also applies syntax highlighting for console output.
+ *
+ * @param statement - The SQLite statement to format
+ * @returns The formatted and colorized SQL statement as a string
```

Comment:
```suggestion
 * @param statement - The SQLite statement to format.
 *
 * @returns The formatted and colorized SQL statement as a string.
```

I commented this on your last PR too. Can you please make sure the format of all the TSDocs is consistent with the rest of our codebase?